ca/certificate-authority.go

```diff
@@ -368,7 +378,9 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(csr x509.CertificateRequest
 		},
 		Serial: serialBigInt,
 	}
-
+	if !ca.forceCNFromSAN {
```

Comment:
This is really for the RA that had a check that made sure SerialNumber was empty in the cert it got back. So, this is for deployability. When the boolean goes away, it should always be set.